packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -262,7 +262,7 @@ class PerformanceMonitoring {
       'checkBrowserResponsiveness'
     )
 
-    if (checkBrowserResponsiveness && !tr.isHardNavigation) {
+    if (checkBrowserResponsiveness && !tr.captureTimings) {
```

Comment:
Would you please add a todo here to refactor `captureTimings` here once the #334 issue is fixed.